server/config.js

```diff
@@ -106,30 +106,41 @@ module.exports = {
       getEnv('ENABLE_STACK_TRACES_ON_ERROR_PAGES', 'false') === 'true',
     useRedisCache: getEnv('ENABLE_REDIS_CACHE', 'true') === 'true',
   },
+  defaultLanguage: 'en',
+  languages: [
+    { lang: 'en', text: 'English' },
+    { lang: 'cy', text: 'Cymraeg' },
+  ],
   sites: {
     berwyn: {
       enabled: false,
       features: [],
+      languages: ['en'],
     },
     bullingdon: {
       enabled: true,
       features: ['incentives', 'money', 'timetable', 'visits'],
+      languages: ['en'],
     },
     cardiff: {
       enabled: true,
       features: ['adjudications', 'incentives', 'money', 'timetable', 'visits'],
+      languages: ['en'],
```

Comment:
Welsh also?

Reply:
If we added Welsh here now, then the language switcher would be available on the Cardiff hub immediately, even though the only currently translated thing would be the title. We need to decide at what point it is worth turning translation on and change this at the appropriate time.

Reply:
Sorry I thought it was feature flagged, my bad. Ideally we'd feature flag it, but you're right, plus it means we can still test the translations via the URL 🙂